cumulusci/core/config/__init__.py

```diff
@@ -27,6 +27,11 @@ class ServiceConfig(BaseConfig):
     pass
 
 
+from cumulusci.core.config.marketing_cloud_service_config import (
+    MarketingCloudServiceConfig,
+)
```

Comment:
Let's not add new aliases to __init__.py files. It means you can't import the package without importing all the modules, which takes longer as the package grows. It can also contribute to import cycles.